src/components/Bookform.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+import Button from './Button';
+
+const Bookform = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // Dispatch the addBook action
```

Comment:
- [  OPTIONAL ] 👩‍💻 "Hello! My advice is that you write code as self-explanatory as possible, even if you need to use some comments. 📝 However, keep in mind that relying too heavily on comments, especially for projects with frequently changing files, may not be the best approach. 🔄 Write code for humans, not just machines. 🧑‍💼
- 🤔 Why? Well, commented code can sometimes become outdated or unclear as the project evolves. 🚀 Your teammates and even your future self may find it challenging to understand and maintain code that relies solely on comments to explain its functionality. 💬
- 👍 So, strive for clean and self-explanatory code that anyone can comprehend at a glance. Use comments sparingly and focus on writing code that tells a clear and coherent story on its own. 📜 This way, your code remains accessible and maintainable, even as your project undergoes frequent changes. 🛠️"